draft-ietf-regext-rdap-geofeed.txt

```diff
@@ -0,0 +1,560 @@
+
```

Comment:
Since I have only worked from .md, not sure how we would keep this .txt update-to-date. Perhaps, use the IETF tools to update it?

Reply:
I was using mmark and xml2rfc to generate the text file, but there's a makefile in the repository now to make it simpler.

Reply:
Cool, will try at some point. :)